en/docs/apis/rest-apis.md

```diff
@@ -27,11 +28,13 @@ including the sample requests and responses.
 - [OIDC Dynamic Client Registration](../../apis/using-the-openid-connect-dynamic-client-registration-rest-apis)
 - [Permission Management](../../apis/permission-management-rest-api)
 - [Personal Information Export](../../apis/using-the-personal-information-export-rest-apis)
+- [SCIM 1.1](../../apis/scim-1.1-apis)
```

Comment:
did we decide to remove this? Just double checking

Reply:
yes